src/test/java/org/testeditor/fixture/host/integrationtest/HostDriverIntegrationTest.java

```diff
@@ -121,4 +122,44 @@ public void typeAtTest() {
         hostDriverFixture.disconnect();
     }
 
+    @Test
+    public void newTransactionTest() {
+        assumeTrue();
+        // given
+        // hostDriverFixture in init
+        String user = System.getenv("USER");
+        String userPwd = System.getenv("USER_PWD");
+        String testSystem = System.getenv("TEST_SYSTEM");
+        String transaction = System.getenv("TRANSACTION");
```

Comment:
given our current discussion we'll probably have to postpone a decision about this test.